### DISCUSSION TITLE
GitHub Campus Program Registration without School-Issued Emails

### DISCUSSION THREAD

**Discussion by eensungkim** [community/community#57490] [GitHub Education]
### Hi everyone,

Our organization is already part of the GitHub Campus Program. But here's the twist: we can't provide school-specific email addresses to our students due to our policies. They need to use their personal email addresses to sign up for the Campus Program.

Now, as it stands, the GitHub Campus Program seems to require a school-issued email for registration. That's a roadblock for our students who are eager to join the program and take advantage of its perks.

I posted about this situation in another discussion ([here](https://github.com/orgs/community/discussions/53829)), and was told to raise a ticket with the support team, which I did. But they pointed me back to you folks here in the community, after waiting for a month for a response to the ticket I raised with the support team ([link](https://support.github.com/ticket/personal/0/2149874))

So, here's my question: I've seen that some institutions (or schools) have managed to have their students join the GitHub Campus Program without a school-issued email. Can we somehow adjust the registration rules or find a workaround to make this happen for our students at Code States as well?

Any ideas or suggestions would be greatly appreciated. Looking forward to your responses!

**Comment by mrepol742** [comment]
It is now mandatory to use school issued email address maybe those who are already in github student pack register before even they mandatory required the school issued email address like the one happened to me...

**Reply by eensungkim** [reply]
I've tried selecting different schools during the registration process, and it seems that a school-issued email isn't mandatory for all of them. Thanks for sharing your experience!

**Comment by mwt** [comment]
Is it against your policies to forward emails at your domain to students' personal accounts? They don't need an inbox to verify the domains.

I know it used to be possible for teachers in the campus program to give students a sign up link to verify them automatically. I don't know why this went away. There have been a lot of changes in the last few years.

**Reply by eensungkim** [reply]
That's quite a workaround! It's worth giving a shot before we get an official answer. Thanks for the great suggestion!

**Reply by eensungkim** [reply]
We've given it a shot with some of our students, but it seems like there's a snag. If an email is already linked with a different GitHub account, registration just won't go through. 😭

**Comment by samus-aran** [comment]
Hi @eensungkim  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍

**Reply by ErwanCarl** [reply]
Hello, 

All my applies to get student github education failed because of the school issued email address , which i don't have, 
I'm an openclassrooms student, and they don't provide us a specific mail, we use our personal ones, 

How can i successfully apply for student github pack ? 

Thanks in advance, 

Best regards, Erwan
